src/credentials_rotators/npm-token-rotation/src/stacks/npm-token-rotation.ts

```diff
@@ -62,7 +62,7 @@ export class NpmTokenRotationStack extends BaseStack {
     );
 
     const deleteOldTokenStateMachine = this.buildTokenDeletionStateMachine(
-      core.Duration.minutes(15),
```

Comment:
Nice! Was the delete token step the one that was timing out?

Reply:
Nope, that was me being confused 😅